zaas-service/src/main/java/org/zowe/apiml/zaas/security/login/zosmf/ZosmfAuthenticationProvider.java

```diff
@@ -76,7 +76,7 @@ public Authentication authenticate(Authentication authentication) {
                     throw new InvalidTokenTypeException("LTPA token in z/OSMF response but configured to expect JWT");
                 }
                 break;
-            default: //AUTO
+            default:
```

Comment:
if JWT is default, then there is no reason to have 2 separate cases, use JWT as default